### ISSUE TITLE
Logue doesn't show in accessibility

### ISSUE THREAD

**Issue by nunoraposo-cloud** [bitwize-ai/Logue#22]
### What happened?

When setup, permissions are granted, but the accessibility fails. This is because the app is not displayed in accessibility. All other permissions work.

### Steps to reproduce

Launch setup, grant permissions.

### Logue version

1.0.0.

### macOS version

macOS 27

### Mac model

MacBook Pro m4

**Comment by shanforge** [comment]
@nunoraposo-cloud  Thanks for the bug report! The issue has been fixed, the PR has been merged, and the fix will be included in the v1.0.1 release, which is scheduled for the end of this week.
